src/main/app/claims/claimStoreClient.ts

```diff
@@ -17,6 +17,18 @@ const claimStoreResponsesApiUrl: string = `${claimApiBaseUrl}/responses/claim`
 
 const logger = Logger.getLogger('claims/claimStoreClient')
 
+function getHeaders (claimant: User, admissionsAllowed: boolean): object {
```

Comment:
I would prefer more `buildCaseSubmissionHeaders` since it is very specific to submission and no other method is using it.